<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>581</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="vL">
      <item>
       <layout class="QHBoxLayout" name="hl1">
        <item>
         <widget class="QLabel" name="l_amount">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Amount:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="dsb_amount">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="l_from">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Currency From:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cb_from"/>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_live">
          <property name="text">
           <string>Live Data (slower)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_convert">
          <property name="text">
           <string>Convert</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="hl2"/>
      </item>
      <item>
       <widget class="QLabel" name="l_selectTo">
        <property name="text">
         <string>Select To Currency:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="lw_toCurr">
        <property name="selectionMode">
         <enum>QAbstractItemView::MultiSelection</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="l_resutlConv">
        <property name="text">
         <string>Result of Conversion:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="te_result"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="hl3">
        <item>
         <widget class="QPushButton" name="pb_exit">
          <property name="text">
           <string>Exit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_reset">
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>654</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pb_exit</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>324</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
